src/lib/api.js

```diff
@@ -37,6 +37,23 @@ export const getItemsFromList = async (token) => {
   const itemsFromList = list.data().items;
 
   if (itemsFromList) {
+    itemsFromList.sort((a, b) => {
+      let fa = a.name.toLowerCase(),
+        fb = b.name.toLowerCase();
+
+      if (fa < fb) {
+        return -1;
+      }
```

Comment:
`else if ` maybe?

Reply:
We have the `return`s and all the cases covered so no need :)

Reply:
yeah, I commented more because I think the code would be more readable with `if/else if` in this case. Since the conditions are related

Reply:
Agreed. Changed.